pkg/api/types.go

```diff
@@ -633,6 +635,8 @@ type EnvVar struct {
 type EnvVarSource struct {
 	// Required: Selects a field of the pod; only name and namespace are supported.
 	FieldRef *ObjectFieldSelector `json:"fieldRef"`
+	// Required: Selects a field of the pod; only name and namespace are supported.
+	ConfigDataRef *ConfigDataSelector `json:"configDataRef"`
```

Comment:
`ConfigDataSelector` is defined in the experimental API. I'm guessing we can't have the stable API reference a type in experimental, can we?